components/test_raftstore/src/util.rs

```diff
@@ -908,6 +908,37 @@ pub fn must_kv_prewrite_with(
     );
 }
 
+pub fn try_kv_prewrite_with(
+    client: &TikvClient,
+    ctx: Context,
```

Comment:
Please set `DiskFullOpt` in `Context`.

Reply:
This option should be set in test case. When not set, it will be the default value.

Reply:
Done